view/frontend/web/template/payment/paypal-button.html

```diff
@@ -0,0 +1,22 @@
+<!-- ko if: (getUsePlaceOrderStyling() == true) -->
+<div class="actions-toolbar rvvup-actions-toolbar" data-bind="style: {display: shouldDisplayPayPalButton() ? 'block' : 'none'}">
+    <div class="primary">
+        <div data-bind="attr: {class: getPaypalBlockStyling()}, style: {display: 'block'
+                ,border :getPaypalBlockBorderStyling(), background :getPaypalBlockABackgroundStyling()}">
+            <div afterRender="afterRenderPaypalComponentProcessor"
+                 data-bind="attr: {'id': getPayPalId()}, css: {disabled: !isPlaceOrderActionAllowed()}">
+            </div>
+        </div>
+    </div>
+</div>
+<!--/ko-->
+
+<!-- ko if: (getUsePlaceOrderStyling() == false) -->
+<!-- PayPal Place Order button, show if PayPal option is selected -->
+<div data-bind="attr: {class: getPaypalBlockStyling()}, style: {display: shouldDisplayPayPalButton() ? 'block' : 'none'
```

Comment:
I think this data-bind will overwrite the class provided?

Reply:
Ah yes, it's set through admin. Removed it.